requirements.txt

```diff
@@ -25,3 +25,8 @@ six==1.12.0
 typed-ast==1.3.1
 urllib3==1.24.1
 wrapt==1.11.1
+gunicorn==19.9.0
+psycopg2==2.7.6.1
+psycopg2-binary==2.7.6.1
+PyJWT==1.6.4
```

Comment:
Good work @wasibani-roy , however `PyJWT` is already added on line 16

Reply:
Thank you @marthamareal I have redone the rebase it should be corrected now